### ISSUE TITLE
Center loss in small batch multiclass training

### ISSUE THREAD

**Issue by SuperbTUM** [KaiyangZhou/pytorch-center-loss#27]
Hello authors and contributors of center loss, thanks for the impressive work. I got a question as I noticed the center update is based on each mini-batch, and if I had a small batch and many classes (way more than batch size), the center update may become tricky, and I wonder if using momentum or other optimizers is necessary for training. Thanks.
